index.js

```diff
@@ -1,41 +1,395 @@
+
+cardsDiv.innerHTML = cardsData.map((item) => {
+  const { placeHolder, cards } = item;
+  const {
+    image, alt, title, description, button, technologies,
+  } = placeHolder[0];
+  const [css, html, bootstrap, ruby] = technologies;
+
+  const { title: title2, description: description2, button: btn } = cards[0];
+
+  return `<div class="placeholder wow fadeInDown">
+          <div class="plchdr_img">
+            <img src=${image} alt=${alt} />
+          </div>
+          <div class="plchdr_text fadeInDown">
+            <h3>${title}</h3>
+            <p class="description">
+              ${description}
+            </p>
+            <ul>
+              <li>${css}</li>
+              <li>${html}</li>
+              <li>${bootstrap}</li>
+              <li>${ruby}</li>
+            </ul>
+            <button type="button" class="bttn">${button}</button>
+          </div>
+        </div>
+       <div class="card card_1" data-scrollmagic="fade-right" data-scrollmagic-offset="300" data-scrollmagic-easing="ease-in-sine" data-scrollmagic-id="card_1">
+
+          <h3 class="card_title">
+            ${title2[0]}<br />
+            ${title2[1]}
+          </h3>
+          <p class="card_description">
+            ${description2}
+          </p>
+          <ul class="tech_stack">
+            <li class="tech">${html}</li>
+            <li class="tech">${bootstrap}</li>
+            <li class="tech">${ruby}</li>
+          </ul>
+          <button class="card_btn bttn" type="button">${btn}</button>
+        </div>
+        <div class="card card_2" data-scrollmagic="fade-right" data-scrollmagic-offset="400" data-scrollmagic-easing="ease-in-sine" data-scrollmagic-id="card_2">
+
+          <h3 class="card_title">
+            ${title2[0]}<br />
+            ${title2[1]}
+          </h3>
+          <p class="card_description">
+            ${description2}
+          </p>
+          <ul class="tech_stack">
+            <li class="tech">${html}</li>
+            <li class="tech">${bootstrap}</li>
+            <li class="tech">${ruby}</li>
+          </ul>
+          <button class="card_btn bttn" type="button">${btn}</button>
+        </div>
+        <div class="card card_3" data-scrollmagic="fade-right" data-scrollmagic-offset="500" data-scrollmagic-easing="ease-in-sine" data-scrollmagic-id="card_3">
+
+          <h3 class="card_title">
+            ${title2[0]}<br />
+            ${title2[1]}
+          </h3>
+          <p class="card_description">
+            ${description2}
+          </p>
+          <ul class="tech_stack">
+            <li class="tech">${html}</li>
+            <li class="tech">${bootstrap}</li>
+            <li class="tech">${ruby}</li>
+          </ul>
+          <button class="card_btn bttn" type="button">${btn}</button>
+          </div>
+          <div class="card card_4" data-scrollmagic="fade-right" data-scrollmagic-offset="300" data-scrollmagic-easing="ease-in-sine" data-scrollmagic-id="card_4">
```

Comment:
While you did a good job trying to work on project requirements you did not implement the cards section to be rendered dynamically using javascript. The only visible things in your app are your introduction page and  your about section not even your contact form is visible.

This is what is displayed when I open the live server:

![Screenshot 2023-04-20 at 2 57 21 PM](https://user-images.githubusercontent.com/79506105/233373209-6b797d07-ceb7-49c3-a15f-09979dfe0f2b.png)

And it might be because of the following error after looking at the console in the dev tools:
![Screenshot 2023-04-20 at 2 56 57 PM](https://user-images.githubusercontent.com/79506105/233373398-d24d1100-816c-4a4f-a965-050c3a9a5735.png)
Please it would be nice if you fixed this issue.


- Secondly I noticed you disabled eslint in your this file please its against microverse policy to disable eslint please kindly fix this by enabling the disabled eslint linters.